docs/orbs-cd.md

```diff
@@ -0,0 +1,69 @@
+# CD Techniques
+
+_Placeholder for more information regarding CD_
```

Comment:
Please change this to: _This is a draft for comment as we continue to develop techniques and features in orb publishing_